docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,106 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available.
+
+| Plan         | High availability features                                                      | Backup history |
+| ------------ | ------------------------------------------------------------------------------- | -------------- |
+| **Hobbyist** | Single node with limited availability                                           | 2 days         |
+| **Startup**  | Single node with limited availability                                           | 2 days         |
+| **Business** | Two nodes (primary + standby: with higher availability)                         | 14 days        |
+| **Premium**  | Three nodes (primary + 2 x standby: with top high availability characteristics) | 30 days        |
+
+## Primary and standby nodes
+
+Aiven's Business and Premium plans offer a primary node and standby nodes.
+A standby service is useful for multiple reasons:
```

Comment:
```suggestion
A standby node is useful for multiple reasons:
```

Reply:
Here I guess we actually mean "service" meaning our standby node offering we have for out customers.

Reply:
Yeah, I understood it but thought it could potentially be confusing with how we use the term "service" within the platform. It's not a separate, distinct service, is it? Not sure if it would be misleading to a user or not.

Reply:
ok, gotcha - since I know the context, it's not misleading for me but if it is to you I guess it might be also misleading for the users. changing to "note"...